shell/machine-config/azure.vue

```diff
@@ -509,7 +531,7 @@ export default {
           :searchable="false"
           :required="true"
           :label="t('cluster.machineConfig.azure.environment.label')"
-          :disabled="disabled"
+          :disabled="true"
```

Comment:
If this input is _never_ editable it can be removed.

Reply:
Deleting here is not appropriate. It needs to be shown to the user to let them know which environment I'm using.